lib/tasks/manageiq/webpacker.rake

```diff
@@ -1,33 +1,34 @@
-if File.exist? 'manageiq-ui-classic.gemspec'
```

Comment:
> This was here to prevent people from trying to run the webpacker tasks (which don't work when run from manageiq/).

If that's the case, then can this move to the manageiq-ui-classic repo and put in the tasks_private directory?  I'm not sure ui-classic was setup with tasks_private, but we did that in the provider gems and it allows those gems to carry rake tasks that can *only* be run from the gem, and are not exposed in manageiq proper.